BuildResidentialHPXML/measure.rb

```diff
@@ -3892,10 +3892,8 @@ def self.set_cooling_systems(hpxml, runner, args)
 
     return if cooling_system_type == 'none'
 
-    if cooling_system_type != HPXML::HVACTypeEvaporativeCooler
-      if args[:cooling_system_cooling_capacity] != Constants.Auto
-        cooling_capacity = args[:cooling_system_cooling_capacity]
-      end
+    if args[:cooling_system_cooling_capacity] != Constants.Auto
```

Comment:
Also need to update the description for the cooling_system_cooling_capacity argument.